modules/apps/frontend-js/frontend-js-clay-sample-web/src/main/resources/META-INF/resources/js/App.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+import Alert from './components/Alert';
+
+import '../css/main.scss';
+
+export default () => {
+	return (
+		<div>
+			<Alert />
```

Comment:
Since this is meant as boilerplate to copy-paste stuff... would it make sense to remove at least the `./components` indirection?

Based on what I understood from the use case, I'd expect people to simply copy-paste React code in here and at most add imports to `clay` components?

If we're actually expecting them to actually create a folder structure, then maybe we're starting to target a different audience?

Reply:
Good point, I'll remove it